src/@openeventkit/event-site/components/MarketingHero/ImagesColumn.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { GatsbyImage, getImage, getSrc } from "gatsby-plugin-image";
+import Slider from "react-slick";
+
+import styles from "./styles.module.scss";
+
+const sliderSettings = {
+  autoplay: false,
+  autoplaySpeed: 5000,
+  infinite: true,
+  dots: false,
+  slidesToShow: 1,
+  slidesToScroll: 1
+};
+
+const ImagesColumn = ({ images }) => {
+  return images.length === 0 ? null : (
+    <div className={`column is-half p-0 ${styles.imagesColumn}`}>
+      {images.length > 1 ? (
+        <Slider {...sliderSettings} >
+          {images.map((image, index) => (
+          <img
+            key={index}
+            src={getSrc(image.src)}
+            alt={image.alt ?? ""}
+          />
+          ))}
+        </Slider>
+      ) : (
+        <GatsbyImage
+          className={`${styles.masonryImage} ${styles.singleImage}`}
```

Comment:
There is already a css class set to that component (singleImage). If you set the styles there, then you don't need to override this file.